walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -65,12 +63,10 @@ else if (request.getAttendees().size() > 0) {
       if (optionalAttendees){
         if (noRequiredMembers && noOptionalMembers){
           continue;
-        }
+        }     
       }
-      else {
-        if (noRequiredMembers){
-          continue;
-        }
+      else if (noRequiredMembers) {
```

Comment:
nit: I believe styleguide prefers to have the `else if` on the same line as the closing bracket for the if
`if (...) {
} else if (...) {
}`

Reply:
Ah didn't see that, will fix it.